.travis.yml

```diff
@@ -19,14 +16,19 @@ branches:
   only:
     - master
     - /^release-.*$/
+    # Matches tags (needed to build documentation)
+    - /^v\d+\.\d+(\.\d+)?(-\S*)?$/
 notifications:
     email: false
-sudo: false
```

Comment:
I don't know what the effect of removing this is.

Reply:
Travis told me it was okay
<img width="1080" alt="image" src="https://user-images.githubusercontent.com/8177701/93158027-78de2680-f75f-11ea-85c4-d855bec696ec.png">

Reply:
sudo: true used to be the default but that changed a few years ago so it's safe to remove